iothub/device/src/Transport/AmqpIoT/AmqpIoTSendingLink.cs

```diff
@@ -194,6 +194,23 @@ internal async Task<AmqpIoTOutcome> SendTwinPatchMessageAsync(string correlation
             return new AmqpIoTOutcome(outcome);
         }
 
+        internal async Task<AmqpIoTOutcome> SubscribeToDesiredPropertiesAsync(string correlationId, TimeSpan timeout)
+        {
+            if (Logging.IsEnabled) Logging.Enter(this, $"{nameof(SubscribeToDesiredPropertiesAsync)}");
+
+            AmqpMessage amqpMessage = AmqpMessage.Create();
+            amqpMessage.Properties.CorrelationId = correlationId;
+            amqpMessage.MessageAnnotations.Map["operation"] = "PUT";
+            amqpMessage.MessageAnnotations.Map["resource"] = "/notifications/twin/properties/desired";
+            amqpMessage.MessageAnnotations.Map["version"] = null;
```

Comment:
Don't you need this for reported?

Reply:
This is the subscribe to desired properties request, what do you mean?

Reply:
I was wondering if you need to subscribe to reported prop as well ? Just like MQTT.

Reply:
The device side only sends reported properties, so I don't understand why we would

Reply:
Offline : Since reported properties are published and get twin has a different sub, we don't need this for reported properties.